.github/workflows/build-all.yml

```diff
@@ -9,23 +9,27 @@ jobs:
     name: ${{ matrix.os-name }}-build
     runs-on: ${{ matrix.os }}
     container: ${{ matrix.container }}
-    env:
-      GHC_VERSION: '9.8.2'
-
-
     defaults:
       run:
         shell: bash
 
     strategy:
       matrix:
-        os: ['windows-latest', 'ubuntu-latest', 'macos-12', 'macos-latest']
+        os: [ 'LinuxARM', 'windows-latest', 'ubuntu-latest', 'macos-12', 'macos-latest']
         include:
+        - os: LinuxARM
+          os-name: Linux-arm
+          project-file: cabal.project.ci.linux
+          ghc: '9.8.2'
+
         - os: ubuntu-latest
           os-name: Linux
           container: fossa/haskell-static-alpine:ghc-9.8.2
           project-file: cabal.project.ci.linux
           ghc: '9.8.2'
+          # Non-Linux environments generally don't need `jemalloc`
+          # and in particular Windows doesn't support it
+          rust-features: jemalloc
```

Comment:
Yes, I need to add to the comment: jemalloc isn't technically supported for musl on arm. 

Initially it seemed to break, I should try it once more because that was many changes ago. In any case, I'll explain it.

Reply:
It looks like it builds and tests are green with jemalloc so I'll leave it on.